Source/FakeItEasy.Tests/Core/ArgumentValueFormatterTests.cs

```diff
@@ -139,7 +141,25 @@ public void Should_prefer_exact_type_formatter_to_interface_formatter()
             var result = this.formatter.GetArgumentValueAsString("string value");
 
             // Assert
-            Assert.That(result, Is.EqualTo("a string"));
+            result.Should().Be("a string");
+        }
+
+        [Test]
+        public void Provided_formatters_should_have_negative_priority()

```

Comment:
Oh, it wasn't a demand or anything!
'sides, I was going to have to force-push anyhow. Might as well rebase while I'm at it.
(This is really just a ploy to get to use [my new toy](https://twitter.com/Hippopottoman/status/693810163329978368).)